### PR TITLE
fix: affinidi supported vc

### PR DESCRIPTION
**Pull request by Vikastc**

### CODE REVIEW

**Comment by Vikastc** [comment]
Error: The property "credentialHash" in the input was not defined in the context.

- getting this error

File: package.json [hunk 1]
```diff
@@ -32,14 +32,23 @@
...
-        "prettier": "3.2.5",
-        "typescript": "^4.8.3"
+        "prettier": "^3.4.1",
+        "typescript": "^5.6.2"
     },
     "dependencies": {
-        "@cord.network/sdk": "0.9.3-1rc20",
-        "moment": "^2.29.4"
+        "@affinidi/tiny-lds-ecdsa-secp256k1-2019": "^1.2.2",
+        "@cord.network/sdk": "0.9.1-1rc18",
```
[the remark this answers is not in the document]
Comment (Vikastc) [reply package.json:46]: No its not working in the latest

File: src/vc.ts [hunk 2]
```diff
@@ -34,6 +34,48 @@ import {
 } from './types';
 
 import { hashContents, calculateVCHash } from './utils';
+import { signCredential } from './affinidi';
+
+const vcTemplate: any = {
+    '@context': [
+        'https://www.w3.org/2018/credentials/v1',
+        {
```
[the remark this answers is not in the document]
Comment (Vikastc) [reply src/vc.ts:42]: Was testing the complete flow with the EcdsaSecp256k1Signature2019 proof by hardcoding few things, but it is failing at jsigs.sign()

**Review by amarts — APPROVED** [review]
LGTM

**Pull request merged**